crypto/ffc/ffc_params_validate.c

```diff
@@ -135,9 +139,14 @@ int ossl_ffc_params_simple_validate(OSSL_LIB_CTX *libctx, const FFC_PARAMS *para
 int ossl_ffc_params_full_validate(OSSL_LIB_CTX *libctx, const FFC_PARAMS *params,
                                   int paramstype, int *res)
 {
+    int tmpres = 0;
+
     if (params == NULL)
         return 0;
 
+    if (res == NULL)
```

Comment:
It looks like you only setup tmpres so that you can do *res without a NULL check in the simple case..
So I don't think it is needed here.

Reply:
I did it here as well to keep the api contract for simple_validate and full_validate the same. So I prefer to keep it there.